src/installer/publish/prepare-artifacts.proj

```diff
@@ -7,10 +7,13 @@
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
   </PropertyGroup>
 
+  <!-- Create all the checksum files. Exclude anything that ends
+       in .nupkg (symbol and regular packages). -->
   <Target Name="CreateChecksums">
     <ItemGroup>
       <ArtifactsForGeneratingChecksums
         Include="@(UploadToBlobStorageFile)"
+        Excldude="*.nupkg"
```

Comment:
I don't think this will work because the nupkgs aren't in the working dir. Maybe use a `Condition` on `%(Extension)` instead, or exclude `@(DownloadedSymbolNupkgFile);@(DownloadedNupkgFile)`?

Reply:
Oh, this has actually already been done:

https://github.com/dotnet/runtime/blob/f2e626060836091c573b4c6d52d5676cc5ef33d6/src/installer/publish/Directory.Build.targets#L26-L29

This PR should be a no-op.

Reply:
Hrmm...that's odd then, because we're getting checksums for the blobs:

https://dev.azure.com/dnceng/internal/_build/results?buildId=563472&view=artifacts&type=publishedArtifacts

Reply:
Oh, that's strange. Later in that file the nupkgs are added back in:

https://github.com/dotnet/runtime/blob/f2e626060836091c573b4c6d52d5676cc5ef33d6/src/installer/publish/Directory.Build.targets#L73

https://github.com/dotnet/runtime/blob/f2e626060836091c573b4c6d52d5676cc5ef33d6/src/installer/publish/Directory.Build.targets#L95

I don't know why we'd be doing that.

Reply:
Looks like it's the ultimate list of things to upload to the parepred artifacts location